src/schema/sources.ts

```diff
@@ -438,6 +442,16 @@ const requireGreaterAccessPrivilege: Partial<
 
 type BaseSourceMember = Pick<SourceMember, 'role'>;
 
+export const hasGreaterAccess = (
```

Comment:
What is the difference with the function below?

Reply:
For this I guess it is not needed anymore. https://github.com/dailydotdev/daily-api/pull/1223#discussion_r1154487100 I'll remove but potentially it will raise again.